tests/integration/scenarios.py

```diff
@@ -134,6 +134,48 @@ def test_remove_virtual_interfaces(self):
         res = subprocess.run(['ip', 'link', 'show', 'dev', 'br54'], capture_output=True, text=True)
         self.assertIn('not exist', res.stderr)
 
+    # https://bugs.launchpad.net/netplan/+bug/2083029
+    def test_netplan_try(self):
+        self.setup_eth(None)
+        with open(self.config, 'w') as f:
+            f.write('''network:
+  renderer: %(r)s
+  ethernets:
+    %(e2c)s:
+      match: {name: %(e2c)s}
```

Comment:
suggestion: Drop the `match:` stanza, as matching on name is implicit (using the NetDefID, e.g. `%(e2c)s`) by default. Same in the YAML below.